app/src/main/java/swati4star/createpdf/model/HomePageItem.java

```diff
@@ -3,15 +3,18 @@
 public class HomePageItem {
 
     private final int mIconId;
+    private final int mDrawableId;
     private final int mTitleString;
 
     /**
      * Constructor for home page item
      * @param iconId - icon drawable id
+     * @param drawableId - drawable id of the Home Page Item
      * @param titleString - title string resource id of home page item
      */
-    public HomePageItem(int iconId, int titleString) {
+    public HomePageItem(int iconId, int drawableId, int titleString) {
```

Comment:
Why is there need of this drawableId?

Reply:
Only one should work. I don't see point of bpth

Reply:
the iconId is not actually holding the actual icon's Id which is used in the home page instead holds the menu Item Id.  I'm not sure but I understand that we can extract the icon's Id from the Menu Item. What do you think?